docs/deploying a sample wildfly application.md

```diff
@@ -0,0 +1,144 @@
+
+1. Create your new application on OpenShift as follows:
+
+        $ oc new-app --name=myapp wildfly:latest~https://github.com/<GitHub username>/os-sample-java-web
+        --> Found image ece19b5 (7 days old) in image stream "wildfly" in project "openshift" under tag "latest" for "wildfly:latest"
+
+            WildFly 10.0.0.Final
+            --------------------
+            Platform for building and running JEE applications on WildFly 10.0.0.Final
+
+            Tags: builder, wildfly, wildfly10
+
+            * A source build using source code from https://github.com/<GitHub username>/os-sample-java-web will be created
+              * The resulting image will be pushed to image stream "myapp:latest"
+              * Use 'start-build' to trigger a new build
+            * This image will be deployed in deployment config "myapp"
+            * Port 8080/tcp will be load balanced by service "myapp"
+              * Other containers can access this service through the hostname "myapp"
+
+        --> Creating resources with label app=myapp ...
+            imagestream "myapp" created
+            buildconfig "myapp" created
+            deploymentconfig "myapp" created
+            service "myapp" created
+        --> Success
+            Build scheduled, use 'oc logs -f bc/myapp' to track its progress.
+            Run 'oc status' to view your app.
+
+1. OpenShift uses `routes` to map domain names to specific applications. OpenShift automatically creates a new internal service, when you create the application, as per the name of the application. The output in Step 4 displays a line, `service "myapp" created`. Expose the name of this service, `myapp` to make your application accessible through the internet and to see it running.
+
+        $ oc expose svc myapp
+        route "myapp" exposed
+
+1. Examine the details of your application as follows:
+
+        $ oc status -v
+        In project My Project (myproject) on server https://192.168.42.205:8443
+
+        http://myapp-myproject.192.168.42.205.xip.io to pod port 8080-tcp (svc/myapp)
+          dc/myapp deploys istag/myapp:latest <-
+            bc/myapp source builds https://github.com/<GitHub username>/os-sample-java-web on openshift/wildfly:latest
+              build #1 running for 4 minutes - 74cdd67: README added (<username> <username@users.noreply.github.com>)
+            deployment #1 waiting on image or update
+
+        Warnings:
+          * dc/myapp has no readiness probe to verify pods are ready to accept traffic or ensure deployment is successful.
+            try: oc set probe dc/myapp --readiness ...
+
+        View details with 'oc describe <resource>/<name>' or list everything with 'oc get all'.
+
+1. You can now access your application by opening the domain `http://myapp-myproject.192.168.42.205.xip.io` displayed in the output above, in the browser of your choice, to see a Hello World message.
```

Comment:
that depends on whether the build has completed. You want to tail the build logs to wait for the completion. Then you know that the app is ready